### DISCUSSION TITLE
GNS3 Vmware VM is missing since 2.2.30

### DISCUSSION THREAD

**Discussion by joseaporo** [GNS3/gns3-gui#3301] [General]
I am unable to download GNS3 VM 2.2.30 or 2.2.31 for VMware ESXi. Clicking "Download" Button on below link for VMware ESXi is broken, Github "Not Found".
https://www.gns3.com/software/download-vm

On https://github.com/GNS3/gns3-gui/releases, I cannot download GNS3.VM.VMware.ESXI.2.2.31.zip, it is not there, the last release which includes it is 2.2.29.

**Comment by grossmj** [comment]
Sorry, something was broken in the release process. I have manually built and uploaded the ESXi VM for v2.2.31 https://github.com/GNS3/gns3-gui/releases/tag/v2.2.31
